src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePickerFlyout.iOS.cs

```diff
@@ -47,6 +47,37 @@ private void InitializeContent()
 			AttachAcceptCommand((DatePickerSelector)Content);
 		}
 
+		#region Content DependencyProperty
+		internal IUIElement Content
+		{
+			get { return (IUIElement)this.GetValue(ContentProperty); }
+			set { this.SetValue(ContentProperty, value); }
+		}
+
+		internal static readonly DependencyProperty ContentProperty =
+			DependencyProperty.Register(
+				"Content",
+				typeof(IUIElement),
+				typeof(DatePickerFlyout),
+				new FrameworkPropertyMetadata(default(IUIElement), FrameworkPropertyMetadataOptions.AffectsMeasure, OnContentChanged));
+		private DatePickerFlyoutPresenter _datePickerPresenter;
+
+		private static void OnContentChanged(object dependencyObject, DependencyPropertyChangedEventArgs args)
+		{
+			var flyout = dependencyObject as DatePickerFlyout;
+
+			if (flyout._datePickerPresenter != null)
+			{
+				if (args.NewValue is IDependencyObjectStoreProvider binder)
+				{
+					binder.Store.SetValue(binder.Store.TemplatedParentProperty, flyout.TemplatedParent, DependencyPropertyValuePrecedences.Local);
```

Comment:
This should not be be needed if the parent of the new content's `DependencyObject.Parent` is set to the flyout, as this will allow for the property inheritance to flow through.

Can you give it a try ?

Reply:
I can do that sure, however do you want this uitest in this beta branch?